CDCgov/prime-reportstream
# feat: convert legacy markdown files into MDX

This PR we're updating the legacy strategy of pulling in markdown content files (.md) in favor of MDX files. Not content is changed.

closes #10392 

Test Steps:
1. Ensure the [Staging Referral Guide page](https://staging.reportstream.cdc.gov/resources/referral-guide) is working as well the [Production Referral Guide Page](https://reportstream.cdc.gov/resources/referral-guide) is working
2.  Ensure the [Staging Getting Started page](https://staging.reportstream.cdc.gov/resources/getting-started-submitting-data) is working as well the [Production Getting Started Page](https://reportstream.cdc.gov/resources/getting-started-submitting-data) is working

## Changes
- Convert MD files to MDX

## Checklist

### Testing
- [x ] Tested locally?
- [x ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [x ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 

### Process

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
